### ISSUE TITLE
mix hex.docs open should automatically fetch docs

### ISSUE THREAD

**Issue by ericmj** [hexpm/hex#290]

**Comment by lexmag** [comment]
I think it should happen thru dialog with user.

**Comment by fteem** [comment]
@lexmag @ericmj I noticed that #292 got merged just as I opened #293. Before, it was quite obvious what the behaviour should be. Right now, I am not so sure.

From my understanding, the command should check if the documentation is present. If not, it should prompt the user if they would like to fetch it locally. Something like:

```
mix hex.docs open phoenix --offline
** (Mix) Documentation file not found, would you like to download it? (Yn)
```

Your thoughts?

**Comment by josevalim** [comment]
@fteem :+1: If the offline documentation is not available we should automatically fetch it.
